R/bibtex.R

```diff
@@ -257,17 +349,18 @@ read.bib <- function(file = findBibFile(package) ,
                       "unknown" = srcfile( file ),
                       srcfile( file, encoding = encoding ) )
 
-    out <- withCallingHandlers(tryCatch(.External( "do_read_bib", file = file,
-                     encoding = encoding, srcfile = srcfile ),
-                       error = function(e){
-                           if(!any(grepl("unprotect_ptr", e)))
-                              stop(geterrmessage(), call. = FALSE)
-                           else
-                              stop("Invalid bib file", call. = FALSE)
-                       }), warning = function(w){
-                             if( any( grepl( "syntax error, unexpected [$]end", w)))
-                               invokeRestart("muffleWarning")
-                           })
+    out <- tryCatch(do_read_bib(
```

Comment:
I needed to change this since C is not used any more. It gives the same messages, there is a test on that